microprofile/6.1/23.0.0.12-TCKResults.adoc

```diff
@@ -0,0 +1,57 @@
+:page-layout: certification
+= MicroProfile 6.1 TCK Results Summary
+
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the MicroProfile TCK results for MicroProfile 6.1.
+
+== Open Liberty 23.0.0.12 MicroProfile 6.1 Certification Summary
+
+MicroProfile 6.1 consists of 8 MicroProfile Specifications plus the Jakarta EE Core Profile 10.0 specfication.
+
+* MicroProfile Config 3.1
+
+* MicroProfile Fault Tolerance 4.0
+
+* MicroProfile Health 4.0
+
+* MicroProfile JWT Authentication 2.1
+
+* MicroProfile Metrics 5.1
+
+* MicroProfile OpenAPI 3.1
+
+* MicroProfile Rest Client 3.0
+
+* MicroProfile Telemetry 1.1
+
+* Jakarta EE Core Profile 10.0
+
+=== Product Name, Version and download URL (if applicable):
+
+https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/runtime/release/23.0.0.12/openliberty-23.0.0.12.zip[Open Liberty 23.0.0.12]
```

Comment:
For 6.0, we point to the microprofile6 zip file instead of the big zip file.  Same with other files.

Reply:
I believe we purposely chose to use the microprofile6 one since it was smaller.  @Emily-Jiang do you remember that as well?